types/lodash/common/function.d.ts

```diff
@@ -840,7 +840,9 @@ declare module "../index" {
          * @param predicate The predicate to negate.
          * @return Returns the new function.
          */
-        negate<T extends (...args: any[]) => any>(predicate: T): T;
```

Comment:
I think you should keep this overload in case someone wants to negate a function that has more than 2 parameters. Or maybe change it to something like:

```
negate(predicate: (...args: any[]) => any)): (...args: any[]) => boolean;
```

We really need variadic kinds (https://github.com/Microsoft/TypeScript/issues/5453) to fix this properly, but this is probably the best we can do without that feature.